packages/@react-native-windows/tester/js/examples-win/DisplayNone/DisplayNoneExample.windows.d.ts

```diff
@@ -0,0 +1,14 @@
+/**
+ * Copyright (c) Microsoft Corporation.
+ * Licensed under the MIT License.
+ * @format
+ */
+/// <reference types="react" />
```

Comment:
You don't need to check these in (.d.ts, .windows.js, .windows.js.map), only the TSX file, and the build would produce the others, so you can remove these from the PR